src/js/utils/DSLUtil.js

```diff
@@ -170,7 +174,13 @@ const DSLUtil = {
         // Properties created through attribute filter will get a boolean value
         //
         case DSLFilterTypes.ATTRIB:
-          memo[prop] = matchingNodes.length === 1;
+          if (matchAgainst.filterParams.text === undefined) {
```

Comment:
Wonder if we should introduce a dedicated symbol e.g., `Symbol("DYNAMIC")` instead of using `undefined` to indicate that we wan't to process _dynamic_ input / filter text.